app/config/config.php

```diff
@@ -77,23 +77,20 @@ function (&$v, $k) {
         }
 
         // Build permission object lists
-        // @todo - convert to tagged services
+        // @deprecated use DI to define permission objects. To be removed in 3.0
```

Comment:
Just a warning that this will conflict with https://github.com/mautic/mautic/pull/8466 for M3 when this makes it there. These changes will need to be applied to https://github.com/mautic/mautic/pull/8466/files#diff-22026a9db7ddaaecd2dedad01f053c7bR1

Reply:
https://github.com/mautic/mautic/pull/8578 adds compatibility for this PR with M3.